packages/flutter_tools/lib/src/ios/ios_workflow.dart

```diff
@@ -72,7 +72,7 @@ class IOSValidator extends DoctorValidator {
   Future<ValidationResult> validate() async {
     final List<ValidationMessage> messages = <ValidationMessage>[];
     ValidationType xcodeStatus = ValidationType.missing;
-    ValidationType brewStatus = ValidationType.missing;
+    ValidationType packageManagerStatus = ValidationType.missing;
```

Comment:
Since this is always set to `ValidationType.installed` below, you can initialize it that way instead.